packages/stark-core/src/configuration/entities/language/language.constants.ts

```diff
@@ -0,0 +1,29 @@
+"use strict";
+import {StarkLanguage} from "./language.entity.intf";
+
+export class StarkLanguages {
+	public static EN_US: StarkLanguage = {
+		isoCode: "en-US",
+		translationKey: "STARK.LANGUAGES.EN",
```

Comment:
We should be careful to not forget to include the translations with the different packages --> #39